integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -82,31 +103,21 @@ export class OpfResourceLoaderService {
    */
   protected loadScript(resource: OpfDynamicScriptResource): Promise<void> {
     return new Promise((resolve, reject) => {
-      const attributes: any = {
+      const attributes: { [key: string]: string } = {
```

Comment:
Can we create type for it in a model if we cannot use: `OpfKeyValueMap`?

Reply:
Mentioned above, we need both:
OpfKeyValueMap is the original format (resource.attributes received from server)
{ [key: string]: string } is format needed for ScriptLoaderService
Thus a conversion is needed.